browser/src/slideshow/StaticTextRenderer.ts

```diff
@@ -137,4 +137,46 @@ class StaticTextRenderer extends TextureAnimationBase {
 	}
 }
 
+class StaticTextRenderer2D {
+	private _canvasContext: RenderContext2d;
+	private _image: ImageBitmap;
+
+	constructor(canvasContext: RenderContext2d) {
+		this._canvasContext = canvasContext;
+	}
+
+	public display(displayText: string) {
+		this._image = this.createImageText(displayText);
+		requestAnimationFrame(this.render.bind(this));
+	}
+
+	public createImageText(displayText: string): ImageBitmap {
+		const width = this._canvasContext.canvas.width;
```

Comment:
Nice to have the static text renderer for the 2d case, too. Anyway shall you avoid code duplication btw `createImageText` and `create2DCanvasWithText` used in the gl case ?